internal/util/paramtable/component_param.go

```diff
@@ -893,6 +893,36 @@ func (p *queryNodeConfig) GetNodeID() UniqueID {
 	return 0
 }
 
+//get config name list
+func (p *queryNodeConfig) GetConfigList() []string {
```

Comment:
use the original key like `queryNode.grouping.topKMergeRatio` is enough, do not need this name converter

Reply:
This was a const of []string, but go could not const a slice, so I use a func instead for ensure it was static. And this work with GetConfigurationByName, could avoid a lot of logical judgment, and if we add or remove some configs will be easier.

Reply:
Actually it's hard to maintain here. Each time a new configuration item is added, this list shall be modified. Even with out manual mistake, it's easy to forget to update this.
As we used reflect package when fetch the value, maybe return a list of reflect.Field may works better?